addons/radar/functions/fnc_canHudBeShown.sqf

```diff
@@ -1,11 +1,11 @@
-params ["_player"];
-!(
-	visibleMap || 
-	{diwako_dui_toggled_off || 
-	{!alive _player || 
-	{(_player getVariable ["ace_spectator_isSet", false]) ||
-	{dialog ||
-	{diwako_dui_ace_hide_interaction && {missionNamespace getVariable ["ace_interact_menu_openedMenuType",-1] > -1} || 
-	{!isnull (missionNamespace getVariable ["ace_arsenal_camera", objNull]) || 
-	{diwako_dui_inFeatureCamera}}}}}}}
-)
+params ["_player"];
+!(
+	visibleMap || 
```

Comment:
This file uses tab indent. Everything else uses 4 space.

Reply:
That is probably my mistake.